src/rp2_common/pico_sd_card/sd_card.c

```diff
@@ -828,6 +828,10 @@ bool sd_scatter_read_complete(int *status) {
         }
         check_crc_count = 0;
     }
+    else if (blocks_complete)
```

Comment:
I wonder if there should be a code-comment that `blocks_complete` will only be set when `sd_scatter_read_complete` returns false? :shrug:

Reply:
I mean, I would've added a comment to the API if the API had any comments at all :rofl: